style.css

```diff
@@ -466,6 +471,61 @@ form {
   display: none;
 }
 
+.menu-mobile {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 90%;
+  background-color: #6070ff;
+  z-index: 1;
+  opacity: 0;
+  visibility: hidden;
+  transition: all 0.3s ease-in-out;
+}
+
+.menu-mobile.active {
+  opacity: 0.9;
+  visibility: visible;
+}
+
+.nav-links-mobile {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  gap: 20px;
+  height: 100%;
+  margin-top: 160px;
+}
+
+.a-links {
+  display: block;
+  font-size: 30px;
+  text-align: center;
+  color: #fff;
+  text-decoration: none;
+}
```

Comment:
- [ ] Please make sure you follow the Figma design🎨 for the menu options here as I've noticed the `font-weight` and `font-size` aren't similar to the given deign below:
- Figma design:
  ![image](https://user-images.githubusercontent.com/104968236/233144688-e029cd98-5367-432a-ba52-8060f6ad0aaa.png)
- Your design:
  ![image](https://user-images.githubusercontent.com/104968236/233144890-1d3eea4b-40b4-4c48-9b66-dddf53a3b3e1.png)